### PR TITLE
Added caching annotations to actually enable cache

### PR DESCRIPTION
**Pull request by hwbllmnn**
@terrestris/devs please review

### CODE REVIEW

File: src/main/java/de/terrestris/shogun/dao/DatabaseDao.java [hunk 1]
```diff
@@ -83,6 +81,9 @@
 import de.terrestris.shogun.model.Group;
 import de.terrestris.shogun.model.MapLayer;
 import de.terrestris.shogun.model.User;
+import de.terrestris.shogun.service.UserAdministrationService;
```
Comment (weskamm) [inline src/main/java/de/terrestris/shogun/dao/DatabaseDao.java]: are those needed?

Comment (hwbllmnn) [inline src/main/java/de/terrestris/shogun/dao/DatabaseDao.java]: Eclipse imports these dynamically also when {@link UserAdministrationService} is used, so javadoc can resolve links.

**Comment by dnlkoch** [comment]
Great work! please merge at will.

Just wondering, is there anything a SHOGun based application needs to be configured with - except Hibernate config `hibernate.cache.use_query_cache` - to enable/disable the cache?

**Comment by hwbllmnn** [comment]
No, if you're happy with the ehcache defaults that's all you need to do (the annotations just do nothing if the cache is disabled per configuration).

**Pull request merged**